### PR TITLE
17 m3 create video controller

### LINKED ISSUE #17: M3: Create video Controller
We need to create the proxy's main controller.
This controller should receive a request with the next format:
http://:/video///

This controller should parse the request call the CDN and return the CDN response.

AC

Video controller should be creathed.
The video controller should work with headers, json and query CMCD request.
The controller should work with dash and hls content.

### PR DESCRIPTION
**Pull request by mcamardon**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ app.use(cors())
 app.use("/", indexRoutes);
```
Comment (JoaquinBCh) [inline src/app.js]: Delete this line too.

File: src/app.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import express from "express";
-
-import indexRoutes from "./routes/index.routes.js";
-import videoTestRoutes from "./routes/video-test.routes.js"
+import videoTestRoutes from "./routes/video-test.routes.js";
+import videoRoutes from "./routes/video.routes.js";
 import cors from 'cors'
+import indexRoutes from "./routes/index.routes.js";
```
Comment (JoaquinBCh) [inline src/app.js]: Delete this line because we should not have the idexRoute any more.

File: src/controllers/index.controller.js
Comment (JoaquinBCh) [inline src/controllers/index.controller.js:1]: Delete the file because we are no using it, and we dont need it.

**Review by JoaquinBCh — CHANGES_REQUESTED** [review]
Delete idex route and controller because we dont need it any more.

**Review by JoaquinBCh — APPROVED** [review]

**Review by coatesjuan — APPROVED** [review]

**Pull request merged**